src/lib/menu/_menu-theme.scss

```diff
@@ -18,6 +18,10 @@
       color: md-color($foreground, 'disabled');
     }
 
+    md-icon {
+      color: md-color($foreground, 'icon');
```

Comment:
It's the color in the spec and in Material 1's version. It looks pretty dramatic without it.